pkg/config/config.go

```diff
@@ -293,6 +294,27 @@ type (
 		SendSMS                     hookConfig `toml:"send_sms"`
 		SendEmail                   hookConfig `toml:"send_email"`
 	}
+	factorTypeConfiguration struct {
+		EnrollEnabled bool `toml:"enroll_enabled"`
+		VerifyEnabled bool `toml:"verify_enabled"`
+	}
+	totpFactorTypeConfiguration struct {
+		factorTypeConfiguration
+	}
+
+	phoneFactorTypeConfiguration struct {
+		factorTypeConfiguration
+		OtpLength    int           `toml:"otp_length"`
+		Template     string        `toml:"template"`
```

Comment:
Is this referring to a template string or path to a template file?

Reply:
This should refer to a Template string. Perhaps can add a comment?

Can change the name of course but the config name in Auth is `Template` so hoping not to deviate.

Open to other suggestions as well

Reply:
Always good to add an example entry to [config.toml](https://github.com/supabase/cli/blob/develop/pkg/config/templates/config.toml#L116), for eg.

```
[auth.mfa.phone]
enroll_enabled = true
...
```